lib/ReactViews/SidePanel/SidePanel.jsx

```diff
@@ -150,19 +150,11 @@ const SidePanel = createReactClass({
             </When>
             <Otherwise>
               <div className={Styles.workbenchEmpty}>
-                <div>Your workbench is empty</div>
-                <p>
-                  <strong>Click &apos;Add data&apos; above to:</strong>
-                </p>
-                <ul>
-                  <li>Browse the Data Catalogue</li>
-                  <li>Load your own data onto the map</li>
-                </ul>
-                <p>
-                  <Icon glyph={Icon.GLYPHS.bulb} />
-                  <strong>TIP:</strong>{" "}
-                  <em>All your active data sets will be listed here</em>
-                </p>
+                <div
+                  dangerouslySetInnerHTML={{
```

Comment:
Even though the defaults are loaded in by terria, we should probably use something like https://github.com/cure53/DOMPurify to sanitise this anyway before we forget (e.g. later down the track it gets loaded in from elsewhere)

Reply:
So I'm wondering how we differentiate between those things which are just text strings vs those which we'll accept a html string for...
Or perhaps as a first cut we should only do strings?